webapp/src/css/inbox.less

```diff
@@ -206,10 +225,12 @@ body {
   }
 }
 
-.filters {
-  height: @filters-bar-desktop-height;
+.tool-bar {
+  display: flex;
+  height: @toolbar-desktop-height;
```

Comment:
There is a "jumping" effect in the topbar
- the top bar height changes
- the elements are displayed higher rather than vertically centered. Note the hamburger icon, and title are vertically centered, while the back chevron icon and title are higher.

Reply:
https://github.com/user-attachments/assets/129c7646-f51d-48a0-9fdc-2a3cb7681651

Reply:
<details>
<summary>Alignment 1 </summary>

<img width="418" alt="Screenshot 2024-09-24 at 11 31 51" src="https://github.com/user-attachments/assets/1011b93f-b032-435e-982a-fa4817232489">

</details>

<details>
<summary>Alignment 2 </summary>

<img width="438" alt="Screenshot 2024-09-24 at 11 32 04" src="https://github.com/user-attachments/assets/bbb02811-71bb-41b0-a75f-cb01eb560618">

</details>


<details>
<summary>Alignment 3 </summary>

<img width="468" alt="Screenshot 2024-09-24 at 11 33 53" src="https://github.com/user-attachments/assets/463bd935-c119-457a-93b0-b7784b74eb64">

</details>

<details>
<summary>Alignment 4 </summary>

<img width="1440" alt="Screenshot 2024-09-24 at 11 35 13" src="https://github.com/user-attachments/assets/f423d5e5-26d4-4134-ad35-88093ccdeab4">

</details>